### PR TITLE
Different replies based on submission type

### LINKED ISSUE #17: Different responses to different template variable values
@xuanxu We've got template variables implemented, and would like different responders for [this template](https://github.com/ropensci/software-review/blob/master/.github/ISSUE_TEMPLATE/A-submit-software-for-review.md) with `<!--submission-type-->Standard<!--end-submission-type-->` and [this template](https://github.com/ropenscilabs/statistical-software-review/blob/main/.github/ISSUE_TEMPLATE/A-submit-software-for-review.md) with `<!--submission-type-->Stats<!--end-submission-type-->`. (The latter template will be added as an additional template to the standard submission repo, so they'll both be templates in the one repo.)

We would like the `add review` command for the first `submission-type` variable (`"Standard"`) to respond as it currently does, but the same command for the second template to generate the same text, but with a different link:

``` rb
respond("#{reviewer} added to the reviewers list. Review due date is #{due_date}. Thanks #{reviewer} for accepting to review! Please refer to [our reviewer guide](https://ropenscilabs.github.io/statistical-software-review-book/pkgreview.html).")
```

### PR DESCRIPTION
**Pull request by xuanxu**
This PR changes the response of the `Reviewers Due Date` responder to include different reviewers guide links for `Standard` and `Stats` submission types.

Closes #17

### CODE REVIEW

**Review by mpadge — APPROVED** [review]
That looks great to me. What do you think @maelle? Note in particular that the additional "Please refer to our reviewer guide" comment is only added for `submission-type` of either `Standard` or `Stats`, while anything else will dump generic reviewer response without that additional link.

We will also need to implement one additional Spanish version for standard submissions (`submission-type` = `Estándar`), but maybe that is best left for a subsequent PR. Note @xuanxu that this value for `submission-type` will change the entire response to a Spanish rather than English version, so have to be re-structured again. Feel free to make an changes here to make those subsequent modifications easier.

I've requested review from @maelle too, so we'll have to wait for that before we can merge. Thanks @xuanxu!!

**Comment by maelle** [comment]
@mpadge 

>  anything else will dump generic reviewer response without that additional link.

Anything else is a malformed submission, correct?

**Review by maelle — APPROVED** [review]
Thank you!!

@mpadge at the moment we do not have Spanish text in the responders at all, the only thing that's been translated is the review template, linked from https://devguide.ropensci.org/reviewerguide.html#general-guidelines.

Therefore it might be best to answer with the same text for "standard" and "estándar" at the moment?

**Comment by mpadge** [comment]
@xuanxu That would then mean changing the current `when: "standard"` to some version of `when: "standard|estandar"`. Is that possible?

**Comment by xuanxu** [comment]
Yes, I'm going to change it to more easily add the spanish responses or other types

File: app/responders/ropensci/reviewers_due_date_responder.rb
Comment (xuanxu) [inline app/responders/ropensci/reviewers_due_date_responder.rb:99]: Done, now adding replies for different submission types is just a matter of adding entries to this list.

**Comment by mpadge** [comment]
Good to merge now then, right @xuanxu?

**Comment by xuanxu** [comment]
Yes, all ready!

**Pull request merged**
